IQKeyBoardManager/IQToolbar/IQUIView+IQKeyboardToolbar.m

```diff
@@ -148,7 +148,16 @@ - (void)addRightButtonOnKeyboardWithText:(NSString*)text target:(id)target actio
     [items addObject:nilButton];
     
     //  Create a done button to show on keyboard to resign it. Adding a selector to resign it.
-    IQBarButtonItem *doneButton =[[IQBarButtonItem alloc] initWithTitle:text style:UIBarButtonItemStyleDone target:target action:action];
+    IQBarButtonItem *doneButton;
+    UIImage *doneImage = [IQKeyboardManager sharedManager].doneImage;
```

Comment:
I don't think we should access [IQKeyboardManager shareManager] here. I don't want any classes to depend on other classes, they should also work separately too.

So Instead of doing this tweak, we can have another method to add a done icon with next/prev and have another method to add done icon only. Also we should check for doneImage in IQKeyboardManager and depending on that we should call appropriate method.